lib/ansible/parsing/vault/__init__.py

```diff
@@ -68,13 +68,8 @@
     from cryptography.hazmat.primitives.kdf.pbkdf2 import PBKDF2HMAC
     from cryptography.hazmat.backends import default_backend
     HAS_PBKDF2HMAC = True
-except ImportError:
+except:
```

Comment:
yeah, if something unexpected happens, you need to know about it.  If it's an optional import, then printing a warning might be the best thing in that case.  So rather than a catch-all that passes you probably should add one for VersionConflict that passes, keep the previous exception handler for DistributionNotFound, and change the raise case in that one into a warning.

It looks like we don't have display available here yet, so you'll need to add an import/definition for that like this code:
https://github.com/ansible/ansible/blob/devel/lib/ansible/cli/playbook.py#L39